be/src/common/config.h

```diff
@@ -87,7 +87,7 @@ CONF_Int32(clear_transaction_task_worker_count, "1");
 // the count of thread to delete
 CONF_Int32(delete_worker_count, "3");
 // the count of thread to alter table
-CONF_Int32(alter_tablet_worker_count, "3");
+CONF_Int32(alter_tablet_worker_count, "6");
```

Comment:
better not changing the default value if it is not that much necessary.

Reply:
ok. maybe it is better decided by user's business scenario, I will restore the value to its original state